platforms/android/library/src/main/java/io/element/android/wysiwyg/inputhandlers/UriContentListener.kt

```diff
@@ -0,0 +1,29 @@
+package io.element.android.wysiwyg.inputhandlers
+
+import android.content.ClipData
+import android.net.Uri
+import android.view.View
+import androidx.core.view.ContentInfoCompat
+import androidx.core.view.OnReceiveContentListener
+
+class UriContentListener(
```

Comment:
If this is only used as a test utility, could you move it into the test directory?

(Or if it is supposed to be exposed as a library utility, then we should add some documentation)